tensorflow/contrib/data/python/kernel_tests/resample_test.py

```diff
@@ -68,6 +71,39 @@ def _testDistribution(self, initial_known):
     returned_dist = class_counts / total_returned
     self.assertAllClose(target_dist, returned_dist, atol=1e-2)
 
+  def testRandomClasses(self):
```

Comment:
Can you reformat this to avoid pylint errors?

Reply:
Done.